haystack/nodes/prompt/prompt_node.py

```diff
@@ -701,7 +701,7 @@ def get_predefined_prompt_templates() -> List[PromptTemplate]:
             "Use the following format:\n\n"
             "Question: the question to be answered\n"
             "Thought: Reason if you have the final answer. If yes, answer the question. If not, find out the missing information needed to answer it.\n"
-            "Tool: [$tool_names]\n"
+            "Tool: pick one of $tool_names \n"
```

Comment:
Whether `[$tool_names]` or `pick one of $tool_names ` works better is hard to decide and I have no strong opinion about that. We would need to run at least some small experiment if we want to optimize these details of the prompt template. I wouldn't focus to much on this optimization at this point. You're suggesting this change because the tool name that the Agent generated for you was in brackets, correct? 
Our reason to use the brackets was because it might be a good way the express that an enumeration of tools follows. Inspiration came from the MRKL implementation [here](https://github.com/hwchase17/langchain/blob/master/langchain/agents/mrkl/prompt.py#L7), which uses
`Action: the action to take, should be one of [{tool_names}]`

Reply:
`pick one of` sounds fair 👍

Reply:
We could still leave the stripping of `[]` in the code although it shouldn't be necessary anymore.

Reply:
yeah, might still be helpful if the user wants to supply a custom `tool_pattern` and the original MRKL `prompt_template`